simpleSequential/isTwoStringEqual.py

```diff
@@ -8,3 +8,5 @@
 #   isTwoStringEqual("Hello", "hello") # return False
 #   isTwoStringEqual("Hello", "Hello") # return True
 #   isTwoStringEqual("", "")           # return true
+def isTwoStringEqual(var1: int, var2: int) -> bool:
```

Comment:
I thought that the problem is “isTwoStringEqual”, the function header say something different.